Backend/app/spotify_electron/user/user_controller.py

```diff
@@ -85,7 +85,9 @@ def get_user(name: str, token: Annotated[TokenData, Depends(JWTBearer())]) -> Re
         user = base_user_service.get_user(name)
         user_json = json_converter_utils.get_json_from_model(user)
 
-        return Response(user_json, media_type="application/json", status_code=HTTP_200_OK)
+        return Response(
```

Comment:
I have seen that the formatter got this Response type of lines wrapped. Maybe its related to line-length on ruff formatter. Imports got out of hand because the long import style( app.x.x.x.) were using, so I decided to increase line length to almost 100chars because nowadays almost everyone has larger screens that the classicals 800x600 that force line length to 80.

Can we check if your formatter is honoring the ruff line length rules? :)

Reply:
I think it is honoring it, it might have wrapped it because I had my vscode workspace split into a few tabs, leaving my editor window to be around ~800x600. I'll look into it and fix it.